persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -705,7 +705,7 @@ builtinGetters = I.fromList
     , (1005,             listOf PersistInt64)
     , (1007,             listOf PersistInt64)
     , (1009,             listOf PersistText)
-    , (143,              listOf PersistText)
+    , (143,              listOf (PersistByteString . unUnknown))
```

Comment:
Interesting.

So, based on what you're saying - the `xml` type shouldn't *ever* have worked, right? So no existing working code should be affected by this change?

Reply:
That's right.